2024/calcite-react/demo/src/Components/Panel/Panel.tsx

```diff
@@ -1,16 +1,21 @@
-import { useEffect, useRef, useState } from 'react';
+import { useEffect, useState } from 'react';
 
+// Import calcite components -  Shell Panel, Block, and Panel
```

Comment:
(optional) this order might read better:

```suggestion
// Import calcite components - Panel, Shell Pane, and Block
```

Reply:
Done